pnpm-lock.yaml

```diff
@@ -13145,7 +13145,7 @@ importers:
         version: link:../../azure-client
       '@fluidframework/azure-client-legacy':
         specifier: npm:@fluidframework/azure-client@^1.2.0
-        version: '@fluidframework/azure-client@1.2.0(encoding@0.1.13)(fluid-framework@1.4.0)'
+        version: '@fluidframework/azure-client@1.2.0(encoding@0.1.13)(fluid-framework@packages+framework+fluid-framework)'
```

Comment:
This is an interesting one. Since the package now has a dependency on the in-repo fluid-framework package too, and that's the one that gets that name in package.json (`fluid-framework@1.4.0` is aliased to `fluid-framework-legacy`), I guess it doesn't seem so crazy that pnpm now thinks this is the version that gets resolved for `fluid-framework` as a peer for `@fluidframework/azure-client-legacy`. It's not precisely _correct_, but wherever this package uses `azure-client-legacy` it should deliberately be using DDS from `fluid-framework-legacy`, so I don't think this represents an immediate problem.

Reply:
Obsolete